stubs/setuptools/setuptools/_distutils/dist.pyi

```diff
@@ -53,7 +54,7 @@ class DistributionMetadata:
 class Distribution:
     cmdclass: dict[str, type[Command]]
     metadata: DistributionMetadata
-    def __init__(self, attrs: Mapping[str, Any] | None = ...) -> None: ...
+    def __init__(self, attrs: Mapping[str, object] | None = ...) -> None: ...
```

Comment:
`Incomplete` might be better here -- I don't really know what's going on in the source code here (and don't really wanna spend time finding out! 😉), but it's probably not actually true that literally any object is okay for the mapping values: https://github.com/pypa/setuptools/blob/9022a215459137d8bd9446d995c0dd159079a2b2/setuptools/_distutils/dist.py#L232-L265.